intermine/bluegenes
# Non-expiring API tokens

## PR authors: @joshkh
### Please describe your PR:

BlueGenes now fetches API tokens that do not expire rather than Session tokens which expire after 24 hours.

To test:

1. Login as usual
2. Open re-frisk
3. Look in `{:auth {:identity {:token ...` and confirm that it matches your API token found in the JSP MyMine section
4. Confirm that you see your email address in the navbar. This is important because the new API token is used to fetch a user's information after authentication.

## Reviewers:
### Review checklist: 

 Before merging, confirm the following tasks have been executed

- [x] review a _minified_ build (e.g. `lein cljsbuild min once` + `lein run`, _not_ `lein figwheel`). 

Checked the following pages load results successfully and allow you to proceed to the results or report page:

- [x] ID resolver + results preview
- [x] Templates execute and show results
- [x] Templates allow you to select lists AND type in identifiers
- [x] Search (dropdown preview version)
- [x] Search (full results page)
- [x] Report page loads, including homologues and tools
- [x] Region search

This is not an exhaustive list - if you spot something else strange please bring it up!
